src/App.tsx

```diff
@@ -62,7 +62,8 @@ wrapHistory(history, {
 const App: FunctionComponent<RouteComponentProps & UserProps> = (props) => {
 	const { tHtml } = useTranslation();
 	const isAdminRoute = new RegExp(`^/${ROUTE_PARTS.admin}`, 'g').test(props.location.pathname);
-	const isPupilUser = String(props?.user?.profile?.userGroupIds?.[0]) === SpecialUserGroup.PupilSecondary;
+	const isPupilUser =
+		String(props?.user?.profile?.userGroupIds?.[0]) === SpecialUserGroup.PupilSecondary;
```

Comment:
Try to use the singular profile_user_group since it only fetches the first group and there should only ever be one group. If we keep this up, we might be able to delete the relationship to the plural groups some day
```suggestion
		String(props?.user?.profile?.profile_user_group?.group?.id) === SpecialUserGroup.PupilSecondary;
```